test/Makefile

```diff
@@ -12,6 +12,17 @@ fake-next-minor-rpm:
 	RPMBUILD_DIR=$(RPMBUILD_DIR) \
 		$(MAKE) -C $(REPO) rpm
 
+.PHONY: local-main-rpm
+local-main-rpm: TMP_REPO !=mktemp -d
+local-main-rpm: RPMBUILD_DIR :=$(REPO)/_output/rpmbuild-main/
+local-main-rpm:
+	git clone --single-branch --branch main --no-hardlinks "$(REPO)" "$(TMP_REPO)"
+	cd "$(TMP_REPO)" && \
+		MICROSHIFT_VERSION="$(MAJOR).$(MINOR).$(PATCH)_main" \
```

Comment:
How hard would it be to include the SHA of main here? Since the clone isn't tied to the PR under review, it may be hard to tell exactly what is being built.

Reply:
Very easy. It's now added to the rpm file names, e.g.
```
_output/rpmbuild-main/RPMS/noarch/microshift-greenboot-4.14.0_main_560fed8c1cbcbc5dda5f091012356a6283a66e4e-1.el9.noarch.rpm
_output/rpmbuild-main/RPMS/noarch/microshift-release-info-4.14.0_main_560fed8c1cbcbc5dda5f091012356a6283a66e4e-1.el9.noarch.rpm
_output/rpmbuild-main/RPMS/noarch/microshift-selinux-4.14.0_main_560fed8c1cbcbc5dda5f091012356a6283a66e4e-1.el9.noarch.rpm
_output/rpmbuild-main/RPMS/x86_64/microshift-4.14.0_main_560fed8c1cbcbc5dda5f091012356a6283a66e4e-1.el9.x86_64.rpm
_output/rpmbuild-main/RPMS/x86_64/microshift-networking-4.14.0_main_560fed8c1cbcbc5dda5f091012356a6283a66e4e-1.el9.x86_64.rpm
```